src/PublicApiGeneratorTests/Assembly_attributes.cs

```diff
@@ -21,6 +21,23 @@ public class Assembly_attributes : ApiGeneratorTestsBase
         [Fact]
         public void Attributes()
         {
+#if NET6_0
+            var api = @"[assembly: PublicApiGeneratorTests.Examples.AttributeWithMultiplePositionalParameters(42, ""Hello"")]
+[assembly: PublicApiGeneratorTests.Examples.AttributeWithNamedParameter(IntValue=42, StringValue=""Hello"")]
+[assembly: PublicApiGeneratorTests.Examples.AttributeWithPositionalParameters1(""Hello"")]
+[assembly: PublicApiGeneratorTests.Examples.AttributeWithPositionalParameters2(42)]
+[assembly: PublicApiGeneratorTests.Examples.Simple]
+[assembly: System.Runtime.InteropServices.ComVisible(false)]
+[assembly: System.Runtime.InteropServices.Guid(""3B8D506A-5247-47FF-B053-D29A51A97C33"")]
+[assembly: System.Runtime.Versioning.TargetFramework("".NETCoreApp,Version=v6.0"", FrameworkDisplayName="""")]
```

Comment:
There's a lot of duplication being introduced here; took me a while to see that the only difference appears to be with the first argument to the `TargetFramework` attribute. Perhaps that value could have been extracted along the following line:

```csharp
#if NET6_0
    const string TFM = ".NETCoreApp,Version=v6.0";
#else
    const string TFM = ".NETCoreApp,Version=v3.1";
#endif

var api = @"... TargetFramework(""${TFM}"", ...";
```
```